CMakeLists.txt

```diff
@@ -333,7 +333,7 @@ if(${CMAKE_SYSTEM_NAME} STREQUAL "Linux")
       set(_icon_filename ${_icon_filename}${_icon_suffix})
     endif()
     install_desktop_files(mantidplot${CPACK_PACKAGE_SUFFIX}.desktop
-                          MantidPlot${_app_name_suffix}
+                          Mantid Plot${_app_name_suffix}
```

Comment:
Do we also want a space between `Plot` and `${_app_name_suffix}`?

Reply:
🤦‍♂ I knew I'd forgotten something - we have this on Windows already